pkgs/development/python-modules/google-auth/default.nix

```diff
@@ -41,6 +42,14 @@ buildPythonPackage rec {
     "google.oauth2"
   ];
 
+  disabledTests = lib.optionals stdenv.isDarwin [
```

Comment:
Did you try ``__darwinAllowLocalNetworking = true;``?

Reply:
https://github.com/NixOS/nixpkgs/pull/116496#discussion_r609399356

Reply:
__darwinAllowLocalNetworking = true; works so I removed the manually disabled commits.